src/solver/conv_asm_implicit_gemm_gtc_bwd.cpp

```diff
@@ -233,22 +252,70 @@ static std::tuple<bool,        // is suitable kernel found
             bool is_gemm_not_empty = gemm_k > 0;
             if(is_gemm_not_empty)
             {
-                if(gemm_k % pConfig->gemm_k_per_block != 0)
+                if(gemm_k % cfg.gemm_k_per_block != 0)
                     gemm_k_valid = false;
             }
         }
         if(!gemm_k_valid)
             continue;
-        // if all valid, break out;
-        break;
+        return std::make_tuple(true,
+                               cfg.GetKernelName(),
+                               cfg.GetBlockSize(),
+                               integer_divide_ceil(gemm_m, cfg.gemm_m_per_block) *
+                                   integer_divide_ceil(gemm_n, cfg.gemm_n_per_block));
     }
-    if(pConfig != tunables.end())
+
+    // second try, try find if packed image size match
+    for(const auto& cfg : tunables)
     {
+        const auto b = cfg.nxe == 0
+                           ? h_tilda_slice * w_tilda_slice
+                           : ((h_tilda_slice * w_tilda_slice + cfg.nxb - 1) / cfg.nxb) * cfg.nxb;
+        const auto gemm_n_packed = n * b;
+        if(cfg.nxe == 0)
```

Comment:
for packed cases, we can simply skip (cfg.nxe==0) , right?

Reply:
Yes. nxe==0 case no pack is support, can make things more simple